RecoTracker/IterativeTracking/python/PixelPairStep_cff.py

```diff
@@ -92,6 +92,8 @@
      originRadius = 0.2,
      fixedError   = 4.
 ))
+from Configuration.Eras.Modifier_highBetaStar_2023_cff import highBetaStar_2023
+highBetaStar_2023.toModify(pixelPairStepTrackingRegions,RegionPSet = dict(originRadius = 0.015))
```

Comment:
I'd add a comment about the issue in the efficiency recovery; once that problem is addressed I expect that `highBetaStar_2018` and `highBetaStar_2023` will be identical.
It may even be more practical to use `(highBetaStar_2018 & run3_upc)` here.

IIRC, the only highBeta run in 2023 was after the BPIX problem appeared, but as I recall those fills had no pixel tracker readout. @sroychow @mmusich please check and correct me if I'm wrong

Reply:
Changed to the approach suggested using "(highBetaStar_2018 & run3_upc)"

Reply:
> IIRC, the only highBeta run in 2023 was after the BPIX problem appeared, but as I recall those fills had no pixel tracker readout. @sroychow @mmusich please check and correct me if I'm wrong

based on [OMS](https://cmsoms.cern.ch/cms/fills/summary?cms_fill_to=9322&cms_fill_from=9000&cms_fill_stableOnly=true) this year we had the fills in the interval 9095-9168 with non-nominal beta* (120cm). All of them happened in era >= Run2023E (which is after the pixel readout incident).
As far as I can tell in none of these fills either Pixel or Strip tracker was in global DAQ excepted fills 9095 and 9128. 
But in any case I don't recall the Tier-0 being moved to the highBetaStar scenario.

Reply:
> once that problem is addressed I expect that highBetaStar_2018 and highBetaStar_2023 will be identical.

Incidentally (and certainly for outside of this particular PR), what's so intrinsically about 2018 of the `highBetaStar_2018` modifier? Can't it just be `highBetaStar` (or `highBetaStar_Phase1` to signal it applies to phase-1 tracking)?

Reply:
`highBetaStar_Phase1` makes sense

2018 is likely the first time it made it to the configs